### PR TITLE
fix(DB/SAI): Teldrassil missing SAI

### PR DESCRIPTION
**Pull request by Knindzagxg**
Started checking lower level zones for missing SAI or other problems.

Using Kiera DB to make new, so mods, feel free to edit SQL if need.

To test:
.tele Teldrassil -> Check creatures around.

Apply SQL and then restart server (Or reload creature template for every ID)

### CODE REVIEW

**Comment by Pondaveia** [comment]
What creatures have you changed btw?

**Comment by Knindzagxg** [comment]
Few spiders and Gnarpine warriors.
Same as for other SQL i made today, they had 0 SAI.

I'm checking lower level zones to see which creature / npc missing it.

**Comment by Pondaveia** [comment]
Thank you 👍

File: data/sql/updates/pending_db_world/rev_1556033550219221000.sql [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+-- Creating a new SmartAI script for [Creature] ENTRY 1998 (name: Webwood Lurker)
+
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 1998;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 1998);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(1998, 0, 0, 0, 0, 0, 100, 0, 2300, 2300, 17600, 17600, 11, 11918, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Webwood Lurker - In Combat - Cast \'11918\'');
```
Comment (poszer) [inline data/sql/updates/pending_db_world/rev_1556033550219221000.sql]: Change [11918](https://wotlk.evowow.com/?spell=11918) to [744](https://wotlk.evowow.com/?spell=744). That is correct ability for this NPC

File: data/sql/updates/pending_db_world/rev_1556033550219221000.sql [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+-- Creating a new SmartAI script for [Creature] ENTRY 2008 (name: Gnarlpine Warrior)
+
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 2008;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 2008);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(2008, 0, 0, 0, 0, 0, 100, 0, 3300, 3300, 15800, 15800, 11, 41056, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Gnarlpine Warrior - In Combat - Cast \'41056\'');
```
Comment (poszer) [inline data/sql/updates/pending_db_world/rev_1556033550219221000.sql]: Change spell [41056](https://wotlk.evowow.com/?spell=41056) to [11976](https://wotlk.evowow.com/?spell=11976).  Strike-11976 is correct ability of this NPC

File: data/sql/updates/pending_db_world/rev_1556033550219221000.sql [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+-- Creating a new SmartAI script for [Creature] ENTRY 2001 (name: Giant Webwood Spider)
+
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 2001;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 2001);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(2001, 0, 0, 0, 0, 0, 100, 0, 3100, 3100, 18200, 18200, 11, 11918, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Giant Webwood Spider - In Combat - Cast \'11918\'');
```
Comment (poszer) [inline data/sql/updates/pending_db_world/rev_1556033550219221000.sql]: Change [11918](https://wotlk.evowow.com/?spell=11918) to [744](https://wotlk.evowow.com/?spell=744). That is correct ability for this NPC

Also if you are able to add 2 more default abilities for this creature:
[Weak Poison](https://wotlk.evowow.com/?spell=6751)
[Web](https://wotlk.evowow.com/?spell=745)

File: data/sql/updates/pending_db_world/rev_1556033550219221000.sql [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+-- Creating a new SmartAI script for [Creature] ENTRY 1999 (name: Webwood Venomfang)
+
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 1999;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 1999);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(1999, 0, 0, 0, 0, 0, 100, 0, 3100, 3100, 18200, 18200, 11, 11918, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Webwood Venomfang - In Combat - Cast \'11918\'');
```
Comment (poszer) [inline data/sql/updates/pending_db_world/rev_1556033550219221000.sql]: Change 11918 to 744. That is correct ability for this NPC

**Comment by Knindzagxg** [comment]
Make sure to check wowdb, not wotlk.evo or any other that's is not fully updated.

**Comment by Knindzagxg** [comment]
Updated.

**Comment by Stoabrogga** [comment]
@Knindzagxg 
This seems not to work:
- Giant Webwood Spider "Weak Poison" on aggro (never got the debuff)
 Could you please look into this?

Successfully tested:
- Webwood Lurker now uses "Poison" in combat
- Gnarlpine Warrior now uses "Strike" in combat
- Timberling Bark Ripper now uses "Pierce Armor" in combat
- Webwood Venomfang now uses "Poison" in combat
- Giant Webwood Spider now uses "Poison" and "Web" in combat

File: data/sql/updates/pending_db_world/rev_1556033550219221000.sql [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+-- Creating a new SmartAI script for [Creature] ENTRY 2001 (name: Giant Webwood Spider)
+
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 2001;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 2001);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(2001, 0, 0, 0, 4, 0, 100, 0, 0, 0, 0, 0, 11, 6751, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Giant Webwood Spider - On Aggro - Cast \'6751\''),
```
Comment (Stoabrogga) [inline data/sql/updates/pending_db_world/rev_1556033550219221000.sql]: Spell 6751 (Weak Poison) cannot be cast directly, please use 6752 (Weak Poison Proc) on the spider instead:
```suggestion
(2001, 0, 0, 0, 11, 0, 100, 0, 0, 0, 0, 0, 11, 6752, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Giant Webwood Spider - On Respawn - Cast \'6752\''),
```

**Pull request merged**